specification/marketplacecatalog/data-plane/Search/preview/2022-08-17-preview/search.json

```diff
@@ -519,6 +519,10 @@
               "x-ms-error-code": {
                 "type": "string",
                 "description": "Error code"
+              },
+              "azure-deprecating": {
+                "type": "string",
+                "description": "Search API version 2022-08-17 will retire on Jan. 31, 2024. After this date, this API version will no longer be supported. To continue using our services, it is essential to migrate to the new Search API preview version 2023-01-01.\nFor detailed guidance, please refer to the updated API documentation, available at: https://learn.microsoft.com/en-us/rest/api/marketplacecatalog/2023-01-01-preview/search/get?tabs=HTTP\nIf you have any questions or require assistance during the transition, our support team is ready to help. Feel free to contact us at MKPL_Platform_API_DL@microsoft.com"
```

Comment:
I assume it is `2022-08-17-preview` get retired, not `2022-08-17`?

Reply:
yes, will fix it. 
@weidongxu-microsoft 
by the way, why the "openapi-pipeline-app / Swagger ApiDocPreview" seems stuck (I cant see the preview of the docs)?

Reply:
It happens sometimes that the CI got stuck. Usually re-run would work...

I see CI is all green now. I've approved the PR. Let me know when you need it merged.

Reply:
Hi  @weidongxu-microsoft,
I don't see any visual notification in the apiPreview informimg of deprecation (where is the text informing of deprecation?).
how can I add such visual deprecation notice? red banner on top with the text I added (see this item created by Product Manager: https://microsoft.visualstudio.com/DefaultCollection/OSGS/_workitems/edit/45554387)

Reply:
The `azure-deprecating` here has nothing to do with deprecation information on API doc page.

This is providing a runtime `azure-deprecating` to your user, so they would see it in logger etc.
https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#deprecating-behavior-notification

Reply:
so what is the common way to add such notification on deprecation?

Reply:
I am not familiar with deprecation procedure. Please check with your PM for deprecation procedure.

If you want to make it clear on MSLearn that certain version will be deprecated, maybe update the page https://learn.microsoft.com/en-us/rest/api/marketplacecatalog/ (it should correspond to a md file in github repo), though I've no idea about red banner. <-- this is only a suggestion, as mentioned, I am not familiar with the procedure

Reply:
@weidongxu-microsoft  I am not sure i understand azure-deprecating header.
we should make the API return such header? or just in the documentation?